### PR TITLE
Color update

### PR DESCRIPTION
**Pull request by felixcanditt**
Thanks for your review :)
I replaced hsl code by hotpink.

### CODE REVIEW

File: client/src/GlobalStyle.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default createGlobalStyle`
   --grey-darkest: hsl(330, 10%, 10%);
   --grey-light: hsl(330, 10%, 85%);
   --grey-lightest: hsl(330, 10%, 96%);
-  --primary: hsl(330,100%,72%);
+  --primary: hotpink;
```
Comment (miriamhaenle) [inline client/src/GlobalStyle.js:11]: Probably the smartes decision you did in the whole project 💪

**Review by miriamhaenle — APPROVED** [review]
👍  awesome job!

File: client/src/GlobalStyle.js (hunk 1 above)
Comment (felixcanditt) [reply client/src/GlobalStyle.js:11]: just one out of 1000 :smirk:

Comment (felixcanditt) [reply client/src/GlobalStyle.js:11]: Thank you so much for your review 💓

**Pull request closed without merging**